pkg/cmd/server/api/v1/conversions.go

```diff
@@ -27,7 +27,7 @@ func SetDefaults_MasterConfig(obj *MasterConfig) {
 			election.LockNamespace = "kube-system"
 		}
 		if len(election.LockResource.Group) == 0 && len(election.LockResource.Resource) == 0 {
-			election.LockResource.Resource = "endpoints"
+			election.LockResource.Resource = "configmaps"
```

Comment:
Yes, updating endpoints all the time causes *all* nodes in a cluster to update iptables (or at least get an event).  Few things watch configmaps.  This was the upstream pivot in 1.7 which I am mimicking.